src/app/directives/response-short.tpl.html

```diff
@@ -0,0 +1,2 @@
+<h4 class="raml-console-resource-param-heading">{{code}}</h4>
+<p markdown="description" class="raml-console-marked-content"></p>
```

Comment:
This could be a good opportunity to create a folder per directive. Right now everything is inside `directive` (both JS and HTML) and the folder is getting really big.